### PR TITLE
fix: cleanup repostiory, migrate to typescript, add refresh command

### LINKED ISSUE #6: How about use fzf instead of mdfind?
As title above, i just want to turn off `md store` totally to save cpu usage.

### PR DESCRIPTION
**Pull request by ChFlick**

### CODE REVIEW

**Review by dubisdev — APPROVED** [review]
LGTM 👍

**Comment by dubisdev** [comment]
I just refactored callbacks into promises to make the code more readable.

Please, check if all this is working ok on MacOS and then we can release a new version 🚀

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cerebroapp/cerebro-mac-apps",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (dubisdev) [inline package.json]: As you introduced the `refresh` command, we should publish a minor. 

```suggestion
  "version": "1.1.0",
```

**Comment by dubisdev** [comment]
I'm not sure what the first `:global` element is doing in [src/Preview/styles.module.css](src/Preview/styles.module.css)

Maybe it is just and old style that is no more used

**Comment by ChFlick** [comment]
@dubisdev mind sending me the npmjs org invitation again? Not sure I got it 🤔

**Comment by dubisdev** [comment]
Of course! I've just sent you a new one 🚀

image

**Pull request merged**